### PR TITLE
yarn global add sizzy

### PR DESCRIPTION
**Pull request by export-mike**
I've exposed sizzy as a npm package. which you can now install globally.

i've tested this locally and did temporarily publish it.

### CODE REVIEW

**Comment by kitze** [comment]
Looks interesting @export-mike, thanks for the PR. What would be the use case for this?

**Comment by export-mike** [comment]
@kitze I can install and run it on my machine, without the need to cd to the cloned directory, 
```
yarn && yarn start
```

I can spin up sizzy whenever I want.

**Comment by export-mike** [comment]
I notice the [Electron branch](https://github.com/kitze/sizzy/tree/feature/electron-app) whats the idea with it?

**Comment by kitze** [comment]
@export-mike the idea with the electron branch is to release Sizzy as a native app, and if in native mode, replace all the <iframe>s with <webview>s which allow embedding of any site, simulating user agents, etc

File: package.json [hunk 1]
```diff
@@ -65,6 +69,7 @@
     "flow": "flow",
     "prettier-all": "prettier --single-quote --bracket-spacing=false --write 'src/**/*.js'",
     "precommit": "lint-staged",
-    "checks": "yarn flow && yarn test"
+    "checks": "yarn flow && yarn test",
+		"prepublish": "yarn build"
```
Comment (export-mike) [inline package.json]: my bad! tabs enabled!

File: package.json [hunk 2]
```diff
@@ -17,6 +20,7 @@
   },
   "dependencies": {
     "director": "^1.2.8",
+    "express": "^4.15.2",
```
Comment (export-mike) [inline package.json]: this is required for the global install to host sizzy with express

**Comment by export-mike** [comment]
I'm closing this, feel free to reopen if you think this PR is useful 👍

**Pull request closed without merging**